<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RefuelingSequence</class>
 <widget class="QDialog" name="RefuelingSequence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tankowanie</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>168</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_refuel">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Tankuj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_finish">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Zakończ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>212</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_fuel_name">
      <property name="text">
       <string>Nazwa Paliwa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLCDNumber" name="lcdNumber_price"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Zł</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLCDNumber" name="lcdNumber_refueling_status"/>
        </item>
        <item>
         <widget class="QLabel" name="label_Liters">
          <property name="text">
           <string>L</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
